extensions/azdata/src/azdata.ts

```diff
@@ -447,8 +447,16 @@ export async function promptForEula(memento: vscode.Memento, userRequested: bool
  * Downloads the Windows installer and runs it
  */
 async function downloadAndInstallAzdataWin32(): Promise<void> {
+	const fileContents = await HttpClient.getTextContent(`${azdataHostname}/${azdataReleaseJson}`);
+	let azdataReleaseInfo;
+	try {
+		azdataReleaseInfo = JSON.parse(fileContents);
+	} catch (e) {
+		throw Error(`failed to parse the JSON of contents at: ${azdataHostname}/${azdataReleaseJson}, text being parsed: '${fileContents}', error:${getErrorMessage(e)}`);
```

Comment:
I'd suggest just having the "failed to parse" string in the error and then log the full message with text in addition. 

Although I guess if we're logging the error then this is fine - but in general I feel like this is too verbose for an error message.